Firestore/tests/System/TransactionTest.php

```diff
@@ -237,4 +237,28 @@ public function testAbort()
         $this->assertTrue($didRetry);
         $this->assertEquals('bat', $this->document->snapshot()['foo']);
     }
+
+    /**
+     * This dataProvider returns the test cases to test how
+     * aggregations work in a transaction.
+     *
+     * The values are of the form
```

Comment:
Better to use `@return` here?

Reply:
We are returning an array of array the nested array having defined variable types. I was wondering how do we put @return here. The return type qualifies more to be array of a defined structure.

I would love to explore this way if possible : ), it would make things really neat.

Reply:
Right now I added variable types before the names so as to make them little intuitive.